pkg/cvo/metrics.go

```diff
@@ -145,15 +146,18 @@ func shutdownHttpServer(parentCtx context.Context, svr *http.Server) {
 	}
 }
 
+const asyncResultHttpsServerName = "HTTPS server"
+
 func startListening(svr *http.Server, tlsConfig *tls.Config, lAddr string, resultChannel chan asyncResult) {
 	tcpListener, err := net.Listen("tcp", lAddr)
 	if err != nil {
-		resultChannel <- asyncResult{name: "HTTPS server", error: err}
+		resultChannel <- asyncResult{name: asyncResultHttpsServerName, error: err}
+		return
```

Comment:
This is not reflected in the commit msg.
I feel we do not need to continue if we hit an error here (`tcpListener` could then be a nil and used later).

Reply:
Nice catch; looks like this snuck in during 0e6136c727b055 (#726)'s [transition to certificate reloading][1].

[1]: https://github.com/openshift/cluster-version-operator/pull/726/commits/0e6136c727b05509c96cd0e6eb135ed7e84d88f2#diff-338242793b0013e033bc403cbd313e2cccfe74d03c9fcc16fa117ac2a735b464L126-R139